logout.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+// Starting the session.
+session_start();
+
+// Unsetting the session variables.
+session_unset();
+// Destroying the session.
+session_destroy();
+
+// Redirecting to the home page.
+header("location: index.php");
+exit;
+
+?>
```

Comment:
Not required.

Reply:
May be it's not necessary but I came across a tutorial on youtube in which the host said that it's a good practice to redirect the user to the index(login/signup) page after destroying the session.